data/language/en-GB.txt

```diff
@@ -3674,6 +3674,8 @@ STR_6357    :{SMALLFONT}{BLACK}Removes all ducks from the map
 STR_6358    :Page {UINT16}
 STR_6359    :{POP16}{POP16}Page {UINT16}
 STR_6360    :{SMALLFONT}{BLACK}{COMMA32}
+STR_6361    :Enable lighting effects on rides (experimental)
+STR_6362    :{SMALLFONT}{BLACK} Rides will be lit up at night.
```

Comment:
I hate to keep moving the goal posts, but the space here is unnecessary. Perhaps it would best be replaced with something more descriptive, anyway:
```suggestion
STR_6362    :{SMALLFONT}{BLACK}If enabled, vehicles for tracked rides will be lit up at night.
```